tests/layer_tests/mo_python_api_tests/test_mo_convert_tf.py

```diff
@@ -367,3 +375,15 @@ def test_mo_import_from_memory_tf_fe(self, create_model, ie_device, precision, i
         if mo_params is not None:
             test_params.update(mo_params)
         self._test_by_ref_graph(temp_dir, test_params, graph_ref, compare_tensor_names=False)
+
+    @pytest.mark.nightly
+    @pytest.mark.precommit_tf_fe
```

Comment:
`@pytest.mark.precommit_tf_fe` is not needed.
` @pytest.mark.precommit` is enough.

Reply:
Removed.